cdc_rsync/error_messages.h

```diff
@@ -21,12 +21,12 @@ namespace cdc_ft {
 
 // Server connection timed out. SSH probably stale.
 constexpr char kMsgFmtConnectionTimeout[] =
-    "Server connection timed out. Please re-run 'ggp ssh init' and verify that "
-    "the IP '%s' and the port '%i' are correct.";
+    "Server connection timed out. Verify that the IP '%s' and the port '%i' "
+    "are correct.";
 
 // Server connection timed out and IP was not passed in. Probably network error.
 constexpr char kMsgConnectionTimeoutWithIp[] =
-    "Server connection timed out. Please check your network connection.";
+    "Server connection timed out. Check your network connection.";
```

Comment:
https://developers.google.com/style/tone#politeness

But actually unused since we now always have an IP. Oh and the name was wrong, should've been kMsgConnectionTimeoutWithOUTIp.